k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml

```diff
@@ -1108,6 +1180,78 @@
     "sha256:e15927e26974af0e066018b056c5f51e514347950e164e4e5be822ed723ee7e0": ["v1.17.10"]
     "sha256:f5de4ad913ffd66ceee261f40fecaec852342168596c9e6d2d8c6fe9f1cd7359": ["v1.18.9"]
     "sha256:f86066539c4a8fb82715e049ef13c0161dbc3b534881b2315476b254b0556733": ["v1.19.1"]
+- name: mock/conformance
```

Comment:
please remove all this mock/*

Reply:
https://github.com/kubernetes/k8s.io/pull/1337/files#diff-36044de293c1633f93272919ab4ce0c69c57831ec1d4517f08ff84fcf694d87eR1183-R1254